web/src/components/profile/ProfileSetup.tsx

```diff
@@ -85,8 +110,34 @@ export default function ProfileSetup(props: Props) {
         completionButtonHref="/studies"
         introText="Complete the following steps to set up your profile and become an approved researcher."
         ariaLabel="Profile setup progress"
+        expiryUrgency={expiryUrgency}
```

Comment:
The `expiryUrgency` is being passed as a prop in 2 different ways here since it is also a property of `profileSetupSteps `. I think it would be good to pass this just once.